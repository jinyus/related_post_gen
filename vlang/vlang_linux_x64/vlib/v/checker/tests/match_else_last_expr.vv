fn main() {
	match 1 {
		1 { println('1') }
		else { println('else') }
		4 { println('4') }
	}
}
