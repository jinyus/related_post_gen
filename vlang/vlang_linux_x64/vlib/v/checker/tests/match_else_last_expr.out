vlib/v/checker/tests/match_else_last_expr.vv:4:3: error: `else` must be the last branch of `match`
    2 |     match 1 {
    3 |         1 { println('1') }
    4 |         else { println('else') }
      |         ~~~~
    5 |         4 { println('4') }
    6 |     }